quicktile/commands.py

```diff
@@ -341,12 +355,17 @@ def move_to_position(winman,       # type: WindowManager
     winman.reposition(win, result, use_rect, gravity=gravity,
             geometry_mask=gravity_mask)
 
+@commands.add('WithBorder', True)
+@commands.add('borderless', False)
```

Comment:
The use of capitals rather than dashes to indicate word boundaries in `WithBorder` is inconsistent with the naming convention for the rest of the commands.

Also, using `borderless` implies that `bordered` isn't a toggle.

Please change these to `bordered-set` and `bordered-unset` so there's no ambiguity and their naming is consistent if I decide to generalize the naming pattern to all of the toggles. (eg. `maximize-set` and `maximize-unset`)

Reply:
Yeah. I wasn't sure about how to name it considering some options were capitalized while some were not.

Reply:
There shouldn't be any. I certainly don't see any capitals in the output of `quicktile --show-actions` on my end.

Reply:
mmm not the actions. But some of the [general ] config are capitalized while some are not. Since I was considering adding "boderless" as a general state, I guess it kind of bled through on my end.